server/graphql/v2/mutation/MemberMutations.js

```diff
@@ -40,6 +47,145 @@ const memberMutations = {
+      account = await fetchAccountWithReference(account, { throwIfMissing: true });
+
+      if (!req.remoteUser.isAdminOfCollective(account)) {
+        throw new Unauthorized('Only admins can edit members.');
+      }
+
+      if (![MemberRoles.ACCOUNTANT, MemberRoles.ADMIN, MemberRoles.MEMBER].includes(args.role)) {
+        throw new Forbidden('You can only edit accountants, admins, or members.');
+      }
+
+      // Make sure we don't edit the role of last admin
+      if (args.role !== MemberRoles.ADMIN) {
+        const admins = await models.Member.findAll({
+          where: {
+            CollectiveId: account.id,
+            role: MemberRoles.ADMIN,
+          },
+        });
+
+        if (admins.length === 1) {
+          if (admins[0].MemberCollectiveId === memberAccount.id) {
+            throw new Forbidden('There must be at least one admin for the account.');
+          }
+        }
+      }
+
+      // Edit member
+      const editableAttributes = pick(args, ['role', 'description', 'since']);
+
+      return models.Member.update(editableAttributes, {
+        where: {
+          MemberCollectiveId: memberAccount.id,
+          CollectiveId: account.id,
+        },
+      });
+    },
+  },
+  removeMember: {
+    type: GraphQLBoolean,
+    description: 'Remove a member from the Collective',
+    args: {
+      memberAccount: {
+        type: GraphQLNonNull(AccountReferenceInput),
+        description: 'Reference to an account of member to remove',
+      },
+      account: {
+        type: GraphQLNonNull(AccountReferenceInput),
+        description: 'Reference to the Collective account',
+      },
+      role: {
+        type: GraphQLNonNull(MemberRole),
+        description: 'Role of member',
+      },
+      isInvitation: {
+        type: GraphQLBoolean,
+      },
+    },
+    async resolve(_, args, req) {
+      if (!req.remoteUser) {
+        throw new Unauthorized('You need to be logged in to remove a member.');
+      }
+
+      let { memberAccount, account } = args;
+
+      memberAccount = await fetchAccountWithReference(memberAccount, { throwIfMissing: true });
+      account = await fetchAccountWithReference(account, { throwIfMissing: true });
+
+      if (!req.remoteUser.isAdminOfCollective(account)) {
+        throw new Unauthorized('Only admins can remove a member.');
+      }
+
+      if (![MemberRoles.ACCOUNTANT, MemberRoles.ADMIN, MemberRoles.MEMBER].includes(args.role)) {
+        throw new Forbidden('You can only remove accountants, admins, or members.');
+      }
+
+      if (args.role === MemberRoles.ADMIN) {
+        const admins = await models.Member.findAll({
+          where: {
+            CollectiveId: account.id,
+            role: MemberRoles.ADMIN,
+          },
+        });
+
+        if (admins.length === 1) {
+          if (admins[0].MemberCollectiveId === memberAccount.id) {
+            throw new Forbidden('There must be at least one admin for the account.');
+          }
+        }
```

Comment:
:ice_cream: Since we're doing that in multiple places, it would be great to have a dedicated helper like:

```ts
isLastAdmin(account, memberAccount): boolean;

if (isLastAdmin(...)) {
  throw new Error(...);
}
```